cfn/cfn.json

```diff
@@ -347,14 +186,9 @@
                     }
                 ],
                 "VpcId": "vpc-d5c5cab0"
-            },
-            "Metadata": {
-                "AWS::CloudFormation::Designer": {
-                    "id": "29d9f23a-24b8-4297-b96c-18e0465ac54f"
-                }
             }
         },
-        "EC2SG3IL8D": {
+        "LiveStreamLoadBalancerSecurityGroup": {
```

Comment:
From an infosec point of view I would suggest restricting this to guardian IP range.
i.e. `"CidrIp": "77.91.248.0/21"`